### ISSUE TITLE
[Feature Request]Alert Notifications via Email/Telegram on Object Detection

### ISSUE THREAD

**Issue by mriamnobody** [sr99622/libonvif#80]
## Feature Description
It would be highly beneficial to include a feature where, upon the detection of predefined object classes (e.g., person), the user receives an alert notification. This notification should include the image/frame where the object was detected. Ideally, users should have the option to receive these alerts through email or Telegram.

## Use Case
This feature would be particularly useful for surveillance and monitoring purposes, allowing users to be promptly informed about important detections without needing to continuously watch the video feed.

## Proposed Implementation
- **Detection Trigger**: When an object of a predefined class is detected, trigger the alert.
- **Notification Content**: Include the timestamp and the image/frame of the detection.
- **Notification Channels**: Allow users to configure their preferred notification channel(s) - Email and/or Telegram.
- **User Configuration**: Provide a simple GUI interface for users to select the object classes they want alerts for and to input their email/Telegram details for notifications.

## Potential Benefits
- **Enhanced Monitoring**: Enables more effective and efficient monitoring by alerting users to significant detections.
- **Versatility**: Useful across a wide range of applications, from home security to wildlife monitoring.

This feature could significantly enhance the utility by integrating real-time alerting capabilities, making it a more powerful tool for surveillance and security.

**Comment by mriamnobody** [comment]
Hey @sr99622, I would like to draw your attention to this feature. This will be a great addon to the Libonvif tool. I see new version of the tool was released yesterday. I'll try and test it soon.

Thank you.

**Comment by sr99622** [comment]
Yes, I agree that this would be a nice feature. After some light research, it seems like there are some python libraries that could do this, maybe even a WhatApp interface could work.

**Comment by mriamnobody** [comment]
At the moment, Libonvif is handling 90% of the tasks related to the specified feature. All that's needed now is a library to send alerts; one option is python-telegram.

**Comment by mriamnobody** [comment]
Hello @sr99622, I have created the said `Alert` Panel in GUI. I would grateful if you could please take a look. The logic to get the detected image is still work in progress. The  ability to test out the `telegram` configuration active in it. I will make PRs
